### ISSUE TITLE
SMTP Mail identification not working with OVH ns0.ovh.com

### ISSUE THREAD

**Issue by ZerooCool** [joomla/joomla-cms#22249]
### Steps to reproduce the issue
Configure SMTP on Joomla, with port 587 and SSL/TLS or STARTTLS, with identification.
(Mail account from OVH, with SMTP server ns0.ovh.com)

### Expected result
Le mail a bien été envoyé.

### Actual result
SMTP Mail not working with 2/3 SMTP.
Option SMTP on Joomla, with port 587 and SSL/TLS or STARTTLS, with identification.

(In french)
Annonce
Impossible de connecter par SMTP https://github.com/PHPMailer/PHPMailer/wiki/Troubleshooting
Erreur
Le test d'e-mail n'a pu être envoyé.

cd /var/log/user.log
Jul 31 17:17:42 : [31-Jul-2018 17:17:42 Europe/Berlin] mail() on [/var/www/html/libraries/vendor/phpmailer/phpmailer/class.phpmailer.php:702]: To: mail@domain.com -- Headers: Date: Tue, 31 Jul 2018 17:17:42 +0200 From: =?utf-8?Q?Green_NRJ_-Alternatives_pour_une_soci=C3=A9t=C3=A9_collaborati?= =?utf-8?Q?ve_et=C3=A9cologique=2E?= mail@domain.com Message-ID: <... ...@... ...> MIME-Version: 1.0 Content-Type: text/plain; charset=utf-8 Content-Transfer-Encoding: 8bit

### System information (as much as possible)
My noob configuration with Debian 9.5 and PHP7.2 and, before, PHP7.0

My light serveur for test : https://github.com/ZerooCool/Linux-Search-Engine-Optimisation/blob/master/installation/3-installer-le-serveur-dans-le-conteneur.sh
My light installation with Joomla (last version) https://github.com/ZerooCool/Linux-Search-Engine-Optimisation/blob/master/installation/4-installer-joomla.sh

### Additional comments

I don't have SSL with my website. You have my config on the top.

2/3 mails SMTP never work **(smtp.visionduweb.com and ns0.ovh.com not work)**

1/3 work. **SMTP with SSL/TLS and Gmail on the 465 port work good**.

I don't use Fail2ban.

I don't have enable “Less Secure Apps” on my **Gmail account and mail work good.**

Mail don't work with 2 other smtp serveur (smtp.visionduweb.com and ns0.ovh.com not work)
I can use this SMTP with redmine, or, mediawiki, and, work good.

For this test, i have use my VPS OVH with Debian 9.5, and, Virtualbox with Debian 9.5.
Exactly the same result, with the same config, 2/3 SMTP don't work.

**Comment by ghost** [comment]
Can one please comment on this Issue? Its open for a Week having Status "New", thanks.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22249">issues.joomla.org/tracker/joomla-cms/22249</a>.</sub>

**Comment by brianteeman** [comment]
This is a configuration issue with ovh
The ovh server ns0.ovh.net is for non ssl traffic - for ssl you should be using ssl0.ovh.net

Please check with your host for further information.

**Comment by ghost** [comment]
Thanks for Answer @brianteeman.

**Comment by ZerooCool** [comment]
I confirm Brian's answer.
STARTTLS + the good SSL server at OVH works. The email is sent.

I used ns0 for redmine and mediawiki on port 587.
Sending mail was running on port 587 with ns0. I do not understand...

I found that when we save the Joomla configuration page, the data is not refreshed without doing F5, so I thought of a Joomla bug for the SMTP connection. The refresh bug has been taken into consideration by the team and seems to have been resolved for the next release.

I confirm that the problem of sending mail is solved, using another SMTP server, directly in SSL: ssl0.ovh.net for OVH.

Thank you Franz and Brian.
